src/lib/assets/icons/Sort.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```

Comment:
Reusable icon for sorting with props that can be used to manipulate the appearance based on the current sort.